graylog2-web-interface/test/setup-jest.jsx

```diff
@@ -12,14 +12,16 @@ registerBuiltinStores();
 
 configure({ adapter: new Adapter() });
 
-const consoleWarn = console.warn;
+const originalWarn = console.warn;
 
 console.warn = jest.fn((...args) => {
   if (!args[0] || !args[0].includes('react-async-component-lifecycle-hooks')) {
-    consoleWarn(format(...args));
+    originalWarn(format(...args));
   }
 });
 
 console.error = jest.fn((...args) => {
-  throw new Error(format(...args));
+  if (!args[0] || !args[0].includes('react-async-component-lifecycle-hooks')) {
```

Comment:
Could you explain a bit more about this change? I don't see how it's related to the scratchpad.